src/template/requirements.txt

```diff
@@ -0,0 +1,3 @@
+jupyter-book
```

Comment:
there is already a requirements.txt file in the main directory. You should use that one.

Reply:
I just removed this and will commit this change after going through all of these.

Reply:
Addressed with commit: https://github.com/IN-CORE/incore-gallery/commit/32199c48c1957392f6b4acccbab72808734bc5ac